resources/META-INF/plugin.xml

```diff
@@ -686,7 +686,9 @@
     <programRunner implementation="io.flutter.run.bazel.BazelRunner"/>
 
     <configurationType implementation="io.flutter.run.bazelTest.FlutterBazelTestConfigurationType"/>
+    <runConfigurationProducer implementation="io.flutter.run.bazelTest.BazelTestConfigProducer"/>
```

Comment:
This line will get deleted during the build unless it is added to the template. This may be the cause of the travis failure.

Reply:
Yup, you'll want to contribute the change to resources/META-INF/plugin_template.xml, and then run `./bin/plugin generate`.

Reply:
Done.